docs/reference/tab-widgets/inference-api/infer-api-task.asciidoc

```diff
@@ -178,6 +178,29 @@ Also, when using this model the recommended similarity measure to use in the
 
 // end::azure-ai-studio[]
 
+// tag::google-vertex-ai[]
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/text_embedding/google_vertex_ai_embeddings <1>
+{
+    "service": "googlevertexai",
+    "service_settings": {
+        "service_account_json": "<service_account_json>", <2>
+        "model_id": "text-embedding-004",
+        "location": "europe-west3", <3>
```

Comment:
Should this also be `<location>` placeholder instead of the concrete value for consistency?